src/Raven.Studio/wwwroot/App/views/shell/about.html

```diff
@@ -149,6 +149,7 @@ <h4>Backups</h4>
                     <div class="name">Encrypted backups</div><div class="value"><i data-bind="attr: { class: licenseAttribute('HasEncryptedBackups') }"></i></div>
                     <h4>ETL</h4>
                     <div class="name">SQL ETL</div><div class="value"><i data-bind="attr: { class: licenseAttribute('HasSqlEtl') }"></i></div>
+                    <div class="name">OLAP ETL</div><div class="value"><i data-bind="attr: { class: licenseAttribute('HasOlapEtl') }"></i></div>
```

Comment:
we need to make sure that will be final license attribute name

Reply:
@grisha-kotler ?